src/main/resources/sentry.properties

```diff
@@ -1 +1,37 @@
-dsn=
+# TODO: https://pihemr.atlassian.net/browse/OBPIH-6417 We can't currently set DSN via openboxes config files such
+#       as application.yml. Once we figure out how to allow that, delete this file from the openboxes repo then add
+#       sentry.properties to the .gitignore. We should be treating this file as if it's an override (like a .env file).
+
+# According to Sentry documentation, this isn't a total Sentry feature switch. Some behaviour may still leak
+# through. To fully disable Sentry, you need to unset the Sentry DSN.
+enabled=true
+
+# If you ever need to debug Sentry changes locally, you can uncomment and set the following properties. Note that
+# they will be overridden by "SENTRY_DSN" and "SENTRY_ENVIRONMENT" environment variables if those are defined.
+#dsn=
+#environment=
+#debug=true
+
+in-app-includes=[ 'org.pih.warehouse' ]
+attach-server-name=true
+enable-auto-session-tracking=true
+
+# Allows additional configuration to be specified via environment variable, such as "SENTRY_DSN" and
+# "SENTRY_ENVIRONMENT" (both of which we require to be configured at runtime for each environment).
+enable-external-configuration=true
+
+# The ratio of all requests to capture for performance tracing. 1.0 is 100% of requests. Disable tracing by setting
+# a value of 0. We keep this number low (as recommended by Sentry) for performance reasons, but it can be increased
+# if we decide we need more data.
+traces-sample-rate=0.1
```

Comment:
this is the setting I was talking about in the tech huddle today with Andy. We only capture 10% of requests. We can adjust this as needed if we're seeing (or not seeing) a performance impact

Reply:
Can we adjust this via Ansible? I assume this file is only available post-deployment, so unless we have a way of making it overridable from openboxes.yml or by adding a sentry.properties override there. Anyway, there might be something that needs to change to make this work i.e. the config locations property defined in openboxes.yml.

Reply:
Good point. 10% is likely a safe bet, but it would definitely be useful to be able to configure this per environment and on the fly.

We could probably set it via environment variable with something like `traces-sample-rate=${SENTRY_SAMPLE_RATE:0.1}`

But I also want to take a look soon at why we're unable to set sentry properties in openboxes.yml. I totally agree that we should give people the option to configure their environments however they want.

Reply:
Yes, that would be awesome. Either setting it in openboxes.yml and/or as a environment variable that openboxes.yml will use to override it's default. 

But this is not a concern for the PR.